test/extensions/filters/network/thrift_proxy/router_test.cc

```diff
@@ -548,7 +548,7 @@ class ThriftRouterTestBase {
   }
 
   void returnResponse(MessageType msg_type = MessageType::Reply, bool is_success = true,
-                      bool is_drain = false) {
+                      bool is_drain = false, bool is_partial_response = false) {
```

Comment:
nit: `is_partial` is probably enough since we know we are talking about responses